pkg/util/test/util.go

```diff
@@ -3107,6 +3115,18 @@ func validateTelemetryStatusInPxctl(telemetryShouldBeEnabled bool, namespace str
 	return nil
 }
 
+func getSecurityAdminToken(namespace string) (string, error) {
+	logrus.Debug("PX Security is enabled, getting token from px-admin-token secret")
+
+	secret, err := coreops.Instance().GetSecret("px-admin-token", namespace)
```

Comment:
does this retry if network issue?

Reply:
I think its enough for it to be inside one retry loop